StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/LinkUI.swift

```diff
@@ -203,16 +218,29 @@ extension LinkUI {
 
 extension LinkUI {
 
-    static let appearance: PaymentSheet.Appearance = {
+    static func applyLiquidGlassIfPossible(configuration: PaymentElementConfiguration) {
+        Self.useLiquidGlass = configuration.appearance.cornerRadius == nil
```

Comment:
`cornerRadius == nil` means "default":
- below iOS 26, the corner radius is 6.0 (ie unchanged from the current default). 
- On iOS 26+,  multiple rounder corner configurations are used to match Liquid Glass design.
```suggestion
        Self.useLiquidGlass = configuration.appearance.cornerRadius == nil, LiquidGlassDetector.isEnabledInMerchantApp
```